### ISSUE TITLE
Can you provide an array of all types implementing an interface on that interface types?

### ISSUE THREAD

**Issue by forabi** [dotansimha/graphql-code-generator#320]
Similar to union types, `possibleTypes` is useful when I want to generate the resolver types for a specific union:

```handlebars
export type UnionMap = {
  {{#each unions}}
  {{ name }}: {
    unionType: {{name}};
    possibleTypeNames: {{#each possibleTypes}}'{{this}}'{{#unless @last}} | {{/unless}}{{/each}};
  }
  {{/each}}
}

```

```ts

export type UnionResolverMap = {
  [UnionName in keyof UnionMap]: {
    __resolveType: UnionOrInterfaceResolver<
      UnionMap[UnionName]['unionType'],
      UnionMap[UnionName]['possibleTypeNames'],
      SchemaContext
    >;
  }
};
```

However, for interface types, this does not seem to be possible currently:

```handlebars
export type InterfaceMap = {
  {{#each interfaces}}
  {{ name }}: {
    interfaceType: {{name}};
    implementorTypeNames: string; // @TODO: improve types
  }
  {{/each}}
}
```

**Comment by dotansimha** [comment]
Done in: https://github.com/dotansimha/graphql-code-generator/pull/339 ;)
@forabi

**Comment by forabi** [comment]
@dotansimha Awesome!

**Comment by dotansimha** [comment]
Done! It's available in 0.9.2! 
@forabi
